Backend/UnitTesting/OperationServiceUT.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ServiceLayer.Services;
+using DataAccessLayer.Models;
+using DataAccessLayer.Database;
+
+namespace UnitTesting
+{
+    /// <summary>
+    /// Summary description for UnitTest1
+    /// </summary>
+    [TestClass]
+    public class OperationServiceUT
+    {
+        OperationService _os;
+        Service service1;
+        Service service2;
+        TestingUtils _ts;
+
+        public OperationServiceUT()
+        {
+            _os = new OperationService();
+            _ts = new TestingUtils();
+        }
+
+        [TestMethod]
+        public void createServiceObject()
+        {
+            service1 = new Service
+            {
+                ServiceName = "" + Guid.NewGuid(),
+                Disabled = false
+                
+            };
+            using (var _db = new DatabaseContext())
+            {
+                Assert.AreEqual(1, _os.createService(service1, _db));
+                Assert.AreEqual(0, _os.createService(service2, _db));

```

Comment:
This could use a comment before these two lines to describe what they're checking.

Reply:
Additionally, these really aren't comprehensive tests. They only check for a success status code, not that the data was actually created within the DB.

Perhaps after creating the services, query the database for the service and check that the service name matches.

If you need an example of this - look at Alfredo's test code.